src/libs/actions/Report.ts

```diff
@@ -4087,6 +4087,16 @@ function markAsManuallyExported(reportID: string, connectionName: ConnectionName
     API.write(WRITE_COMMANDS.MARK_AS_EXPORTED, params, {optimisticData, successData, failureData});
 }
 
+function subscribeToActiveGuides() {
+    Pusher.subscribe(`activeGuides`).catch((error: ReportError) => {
```

Comment:
I think we need to declare the channel as constant - 

It should be - `presence-activeGuides${CONFIG.PUSHER.SUFFIX}`
We can make `presence-activeGuides` as a constant

Reply:
@daledah please take a look at these comments when you get a moment